docs/guilherme.martins/modulo_1/styles.css

```diff
@@ -41,7 +41,11 @@ body {
   width: 800px;
   margin: 0 auto;
   background-color: #131313;
+<<<<<<<< HEAD:docs/guilherme.martins/modulo1/styles.css
+  padding: 0 20px ;
+========
   padding: 25px;
+>>>>>>>> 0156ff32ab2324130d6f9e16eb5a20585f6ed42b:docs/guilherme.martins/modulo_1/styles.css
```

Comment:
nit, não precisa alterar o módulo.
Referencia: https://www.atlassian.com/git/tutorials/resetting-checking-out-and-reverting
```
git pull upstream (branch principal)
git checkout upstream/master docs/guilherme.martins/modulo_1/styles.css
```